Buildings/Controls/OBC/ASHRAE/G36/FanCoilUnit/Controller.mo

```diff
@@ -0,0 +1,1159 @@
+within Buildings.Controls.OBC.ASHRAE.G36.FanCoilUnit;
+block Controller
+  "Fan coil unit controller that comprises subsequences for controlling fan speed and supply air temperature"
+
+  parameter Buildings.Controls.OBC.ASHRAE.G36.Types.CoolingCoil cooCoi=Buildings.Controls.OBC.ASHRAE.G36.Types.CoolingCoil.WaterBased
+    "Cooling coil type"
+    annotation (__cdl(ValueInReference=false),
+                Dialog(group="System and building parameters"));
+  parameter Buildings.Controls.OBC.ASHRAE.G36.Types.HeatingCoil heaCoi=Buildings.Controls.OBC.ASHRAE.G36.Types.HeatingCoil.WaterBased
+    "Heating coil type"
+    annotation (__cdl(ValueInReference=false),
+                Dialog(group="System and building parameters"));
+
+  parameter Boolean have_winSen=false
+    "True if the zone has window status sensor";
+
+  parameter Boolean have_occSen=false
+    "True if the zone has occupancy sensor";
+
+  parameter Real heaDea(
+    unit="1",
+    displayUnit="1")=0.05
+    "Heating loop signal limit above which controller operation changes from deadband mode to heating mode"
+    annotation (Dialog(
```

Comment:
@karthikeyad-pnnl add `__cdl(ValueInReference= xxx)` annotations. Please see for example the multizone VAV AHU controller sequences. This annotations should be added to the subsequences as well.

Reply:
@JayHuLBL Ok. We will try to propagate these additional annotations by early next week. Will reach out if I need any guidance